### PR TITLE
Docker login before make build step

### PR DESCRIPTION
**Pull request by Calebjh**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,16 +10,18 @@ jobs:
     # Latest image build/push
     - if: branch = master
       script:
+        - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
         - make latest && make test
       deploy:
         provider: script
         script: echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin && make push-latest
```
Comment (kd7lxl) [inline .travis.yml:17]: This is redundant (I think... Travis doesn't document this scope issue well)

**Review by kd7lxl — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (kd7lxl) [inline .travis.yml:17]: For compatibility with PRs from forks, you'll need to conditionally login.
https://docs.travis-ci.com/user/pull-requests/#pull-requests-and-security-restrictions

**Comment by Calebjh** [comment]
Replaced by #80

**Comment by kd7lxl** [comment]
This PR was a good test because we still need the PR test to work on forks. We just don't need it to push the image on PRs. The job needs to be modified so it only pushes on not PRs.

**Pull request closed without merging**